index.html

```diff
@@ -191,18 +237,28 @@ <h2 class="aboutme-card-title">Skills</h2>
     <!--end about me section-->
 
       <!--start of footer section / contact form-->
-      <footer>
-      <form  method="post" action="https://formspree.io/f/xyyowwnj">
-        <section class="contact-form-container">
+    <section class="contact-form-container"> 
         <h2 class="contact-form-title">
-                I&#39;m always interested in hearing about new projects, so if you&#39;d like to chat please get in touch.</h2>
-        <ul>
-            <li><input type="text" placeholder="Full name" required></li>
-            <li><input type="email"  placeholder="Email address" required></li>
-            <li><textarea name="textarea"  maxlength="500" required>Enter text here</textarea></li>
-            <li><input class="btn-green" type="submit" value="Get in touch"></li>
-            <li>
-                <ul class="contact-form-social-icon">
+            I&#39;m always interested in hearing about new projects, so if you&#39;d like to chat please get in touch.
+        </h2>
+      <form  class="contact-form-desc" method="post" action="https://formspree.io/f/xyyowwnj">
+            <ul>
+                <li><input id="full-name" type="text" placeholder="Full name" required></li>
+                <li>
+                    <ul class="f-l-name">
+                        <li><input id="first-name" type="text" placeholder="First name" required></li>
+                        <li><input id="last-name" type="text" placeholder="Last name" required></li>
+                    </ul>
+                </li>
```

Comment:
- [ ] [OPTIONAL] Please, consider setting `maxlength = 30` for inputs, to avoid randomly entering big-length words from your webpage visitor.